net/instaweb/rewriter/render_blocking_html_computation.cc

```diff
@@ -100,8 +100,11 @@ void RenderBlockingHtmlComputation::Compute(const GoogleString& url) {
   }
 
   bool is_authorized = false;
+  // Note: this is for HTML, so may be kReconstruction would be appropriate,
+  // but it's tough to audit since this doesn't seem to have any users?
```

Comment:
Did something different: If I comment out render_blocking_html_computation.cc and render_blocking_html_computation_test.cc in their appropriate .gyp files the whole thing compiles & links & runs unit tests just fine.

I think this was probably used by the mobilizer, shall I remove it?